src/Utils.cc

```diff
@@ -756,14 +756,14 @@ std::pair<K*, V*> radix_sort_parallel(
 #ifdef _MSC_VER
   const size_t array_size = (size_t)RDX_HIST_SIZE * maxthreads;
   // fixes MSVC error C2131
-  int* const histogram = static_cast<int*>(
-      fbgemm::fbgemmAlignedAlloc(64, (array_size) * sizeof(int)));
-  int* const histogram_ps = static_cast<int*>(
-      fbgemm::fbgemmAlignedAlloc(64, (array_size + 1) * sizeof(int)));
+  int64_t* const histogram = static_cast<int64_t*>(
+      fbgemm::fbgemmAlignedAlloc(64, (array_size) * sizeof(int64_t)));
+  int64_t* const histogram_ps = static_cast<int64_t*>(
+      fbgemm::fbgemmAlignedAlloc(64, (array_size + 1) * sizeof(int64_t)));
 
 #else
-  alignas(64) int histogram[RDX_HIST_SIZE * maxthreads];
-  alignas(64) int histogram_ps[RDX_HIST_SIZE * maxthreads + 1];
+  alignas(64) int64_t histogram[RDX_HIST_SIZE * maxthreads];
+  alignas(64) int64_t histogram_ps[RDX_HIST_SIZE * maxthreads + 1];
```

Comment:
same here

Reply:
I cannot use auto here, as this is a declaration without initialization.